src/content/en/updates/2019/05/chromium-chronicle-2.md

```diff
@@ -0,0 +1,110 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Chromium Chronicle, a monthly series geared specifically to Chromium developers — the developers who build the browser.
+
+{# wf_updated_on: 2019-05-14 #}
+{# wf_published_on: 2019-05-21 #}
+{# wf_tags: chromium-chronicle #}
+{# wf_featured_image: /web/updates/images/generic/cr-chron.jpg #}
+{# wf_featured_snippet: Flaky tests are a common problem in Chrome. They impact the productivity of other developers, and get disabled over time. This month, we take a look at how to fight test flakiness. #}
+{# wf_blink_components: N/A #}
+
+<style>
+  body:not(.devsite-dark-code) pre.prettyprint.cc-bad {
+    background-color: #fff7f7;
+  }
+  body:not(.devsite-dark-code) pre.prettyprint.cc-good {
+    background-color: #f7fff7;
+  }
+</style>
+
+# The Chromium Chronicle: Fighting Test Flakiness {: .page-title }
+
+**Episode 2:** May 2019
+
+*by Vasilii in Munich*
+
+[Flaky Tests][flaky-tests-context]are a common problem in Chrome. They
+impact the productivity of other developers, and get disabled over time. 
+Disabled tests mean diminishing test coverage. 
+
+**Triaging Stage**
+
+**The OWNERS of directories are responsible for fixing their flaky tests.**
+If you received a bug about a flaky test, spend a few minutes and
+**comment what went wrong** on the bug. If you have an old flaky test and
+it's unclear what went wrong, **try to simply re-enable** the test.
+**Reassign the bug ASAP** if it's clearly a problem in another component.
+The owners of that component should have better judgement about the failure, 
+
+**Debugging Stage:**
```

Comment:
This should probably be an H2:
```
## Debugging stage
```

Note, it's sentence case and doesn't include the colon at the end.